docs/content/direct/user-guide.md

```diff
@@ -31,3 +31,59 @@ Additionally, the recipe for registering a WEC with the ITS needs to be modified
 ## When everythig is not on the same machine
 
 Thus far we can only say how to handle this when the hosting cluster is OpenShift. The problem is getting URLs that work from everywhere. OpenShift is a hosted product, your clusters have domain names that are resolvable from everywhere. In other words, if you use an OpenShift cluster as your hosting cluster then this problem is already solved.
+
+## Rule-based customization
+
+KubeStellar can distribute one workload object to multiple WECs, and it is not uncommon for users to need some customization to each WEC. By _rule based_ we mean that the customization is not expressed via one or more literal expressions but rather can refer to _properties_ of each WEC by property name. As KubeStellar distributes or transports a workload object from WDS to a WEC, the object can be transformed in a way that depends on those properties.
+
+KubeStellar currently has a simple but limited way to specify rule-based customization, called "template expansion". This works with templates as defined in the "text/template" standard package of Go, and is applied to each leaf string of an object independently. More expressive customization will be developed in the future.
```

Comment:
I would remove the sentence `More expressive customization will be developed in the future.`
it doesn't add any information. if and when extending the customization we can describe it (as any other feature).

Reply:
Reworded.